From 8f7f553d89bf0a4f6b7a342539283d712c9a46bf Mon Sep 17 00:00:00 2001
From: Carlos Rodriguez <carlos.a.rodriguez@intel.com>
Date: Fri, 22 Apr 2016 08:17:44 -0500
Subject: [PATCH 2/4] ASoC: rt5645: Increase micbias output voltage.

In order to support audio recording with headsets which output
voltage is bigger than 2.0V, the micbias output voltage should
be increased. Current value is 1.8V. Therefore change the micbias
output voltage to 2.7V.

Change-Id: I7cc7b19a2ea3710918570731a79fd5afbac0032a
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-29994
Signed-off-by: Carlos Rodriguez <carlos.a.rodriguez@intel.com>
Reviewed-on: https://android.intel.com:443/498755
---
 sound/soc/codecs/rt5645.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/rt5645.c b/sound/soc/codecs/rt5645.c
index 0b39a82..ddcc02d 100644
--- a/sound/soc/codecs/rt5645.c
+++ b/sound/soc/codecs/rt5645.c
@@ -72,7 +72,7 @@ static struct rt5645_init_reg init_list[] = {
 
 #ifdef JD1_FUNC
 	{ RT5645_IRQ_CTRL2	, 0x0200 },
-	{ RT5645_MICBIAS	, 0xC008 },
+	{ RT5645_MICBIAS	, 0x0008 },
 	{ RT5645_GEN_CTRL3	, 0x1180 },
 	{ RT5645_CJ_CTRL3	, 0x4000 },
 #endif
-- 
1.9.1

